router/gin/endpoint.go

```diff
@@ -61,6 +61,8 @@ func CustomErrorEndpointHandler(configuration *config.EndpointConfig, prxy proxy
 					c.Writer.Header().Add(k, v)
 				}
 			}
+
+			c.Status(response.Metadata.StatusCode)
```

Comment:
`response` could be nil

Reply:
@kpacha the line https://github.com/donus3/krakend/blob/16ea2a68e1f8f287de3b041b52372eb9a786b297/router/gin/endpoint.go#L51 already check `if response != nil`

Reply:
Btw, I will check `if response.Metadata != nil`

Reply:
@kpacha pls reopen